packages/jazz-tools/src/auth/PassphraseAuth.ts

```diff
@@ -20,12 +23,25 @@ import { AuthSecretStorage } from "./AuthSecretStorage.js";
 export class PassphraseAuth {
   passphrase: string = "";
 
+  private crypto: CryptoProvider;
+  private authenticate: AuthenticateAccountFunction;
+  private register: RegisterAccountFunction;
+  private authSecretStorage: AuthSecretStorage;
+  public wordlist: string[];
+
   constructor(
-    private crypto: CryptoProvider,
-    private authenticate: AuthenticateAccountFunction,
-    private authSecretStorage: AuthSecretStorage,
-    public wordlist: string[],
-  ) {}
+    crypto: CryptoProvider,
+    authenticate: AuthenticateAccountFunction,
+    register: RegisterAccountFunction,
+    authSecretStorage: AuthSecretStorage,
+    wordlist: string[],
+  ) {
+    this.crypto = crypto;
+    this.authenticate = authenticate;
+    this.register = register;
+    this.authSecretStorage = authSecretStorage;
+    this.wordlist = wordlist;
+  }
```

Comment:
why not just
```ts
  constructor(
    private crypto: CryptoProvider,
    private authenticate: AuthenticateAccountFunction,
    private register: RegisterAccountFunction,
    private authSecretStorage: AuthSecretStorage,
    public wordlist: string[],
  ) {}
```
to remove some noise?

Reply:
This is one of the Typescript features with runtime code and the new direction for Typescript now that has become part of Node.js is to start to deprecate this kind of features.

It's a very long process and it's not a solid reason to stop using it, but I've done this change just to see how bad it looks to go back and use the "manual" initializers.

I can revert the change, kind-of agree that looks quite verbose compared to before.

Reply:
makes sense, in general i'd leave "style" changes isolated to their own PRs (and possibly added to .git-blame-ignore-revs), but not a biggie. 🐑 👽